service/src/test/java/greencity/service/UBSClientServiceImplTest.java

```diff
@@ -1157,6 +1209,107 @@ void testSaveToDBfromIForIFThrowsException() throws InvocationTargetException, I
         });
     }
 
+    @Test
+    void testSaveToDBfromIFThrowsException() throws InvocationTargetException, IllegalAccessException {
```

Comment:
Could you rename this test, please? Because from context it is not clear what you are testing